### PR TITLE
Update locale.js

### PR DESCRIPTION
**Pull request by shenweiyan**
Improve the new version of locale.js Chinese information.

### CODE REVIEW

**Comment by bgruening** [comment]
@shenweiyan thanks a lot!

Do you think those improvements make sense?

65 "Copy": "复制",

67 "Extract Workflow": "提取工作流程",

162 "This dataset is currently uploading": "您的数据集正在上传中",

File: client/src/nls/zh/locale.js [hunk 1]
```diff
@@ -180,54 +187,55 @@ define({
...
+    "Download local tool": false,
     "Tool lineage": false,
-    "Reload a tool's configuration": "重新加载工具的配置",
-    "Review tool migration stages": "查看工具迁移状态",
-    "View Tool Error Logs": "查看工具错误日志",
-    "Manage Allowlist": "管理显示白名单",
-    "Manage Tool Dependencies": "管理工具依赖",
-    Users: "用户",
-    Groups: "组别",
+    "Reload a tool's configuration": false,
```
Comment (bgruening) [inline client/src/nls/zh/locale.js:229]: @shenweiyan is there a reason you removed this translation?

Comment (shenweiyan) [reply client/src/nls/zh/locale.js:229]: As an administrators, I think the English interface will be better. And i18n processing is just to make it more convenient for non-administrators to use galaxy.

**Review by shenweiyan — COMMENTED** [review]
> @shenweiyan thanks a lot!
> 
> Do you think those improvements make sense?
> 
> 65 "Copy": "复制",
> 
> 67 "Extract Workflow": "提取工作流程",
> 
> 162 "This dataset is currently uploading": "您的数据集正在上传中",

Thanks for the reminder, it should be make no sense.

**Comment by mvdbeek** [comment]
Wonderful, this is always appreciated @shenweiyan!

**Pull request merged**